src/web/wms/src/main/java/org/geoserver/wms/web/publish/WMSLayerConfig.java

```diff
@@ -1,161 +1,237 @@
-                new Palette<StyleInfo>(
-                        "extraStyles", stylesModel, styles, new StyleNameRenderer(), 10, false) {
-                    private static final long serialVersionUID = -3494299396410932090L;
-
-                    /** Override otherwise the header is not i18n'ized */
-                    @Override
-                    public Component newSelectedHeader(final String componentId) {
-                        return new Label(
-                                componentId,
-                                new ResourceModel("ExtraStylesPalette.selectedHeader"));
-                    }
-
-                    /** Override otherwise the header is not i18n'ized */
-                    @Override
-                    public Component newAvailableHeader(final String componentId) {
-                        return new Label(
-                                componentId,
-                                new ResourceModel("ExtraStylesPalette.availableHeader"));
-                    }
-                };
-        extraStyles.add(new DefaultTheme());
-        styleContainer.add(extraStyles);
-
-        TextField<Integer> renderingBuffer =
-                new TextField<Integer>(
-                        "renderingBuffer",
-                        new MapModel(new PropertyModel(layerModel, "metadata"), LayerInfo.BUFFER),
-                        Integer.class);
-        renderingBuffer.add(RangeValidator.minimum(0));
-        styleContainer.add(renderingBuffer);
-
-        add(new TextField<String>("wmsPath", new PropertyModel<String>(layerModel, "path")));
-
-        List<WMSInterpolation> interpolChoices = Arrays.asList(WMSInterpolation.values());
-
-        PropertyModel<WMSInterpolation> defaultInterpolModel =
-                new PropertyModel<WMSInterpolation>(layerModel, "defaultWMSInterpolationMethod");
-        DropDownChoice<WMSInterpolation> interpolDropDown =
-                new DropDownChoice<WMSInterpolation>(
-                        "defaultInterpolationMethod",
-                        defaultInterpolModel,
-                        interpolChoices,
-                        new InterpolationRenderer(this));
-        interpolDropDown.setNullValid(true);
-        add(interpolDropDown);
-    }
-
-    private class InterpolationRenderer extends ChoiceRenderer<WMSInterpolation> {
-
-        private static final long serialVersionUID = 4230274692882585457L;
-
-        private Component parent;
-
-        public InterpolationRenderer(Component parent) {
-            this.parent = parent;
-        }
-
-        @Override
-        public Object getDisplayValue(WMSInterpolation interpolationMethod) {
-            return new StringResourceModel(interpolationMethod.name(), parent).getString();
-        }
-
-        @Override
-        public String getIdValue(WMSInterpolation object, int index) {
-            return object.name();
-        }
-    }
-}
+/* (c) 2014 - 2016 Open Source Geospatial Foundation - all rights reserved
```

Comment:
What happen here? Is impossible for me to understand what new or old.

Reply:
this is eclipse , I will fix it